mkdocs.yml

```diff
@@ -47,5 +47,5 @@ nav:
 extra:
   fleet:
     apiVersion: fleet.cattle.io/v1alpha1
-    version: v0.3.0
+    version: v0.3.3
     helmversion: 0.3.0
```

Comment:
Wondering if we have to worry about `helmversion`. CC: @StrongMonkey

Reply:
yeah we should update this too. @miguelpoyatosmora Can you also update it

Reply:
Oh, thanks! what about this?

Reply:
Thanks! LGTM now